bubble_sort.rb

```diff
@@ -0,0 +1,31 @@
+def bubble_sort(array)
+  array.length.times do
+    (0...(array.length - 1)).each do |i|
+      if array[i] > array[i + 1]
+        deleted = array.delete_at(i)
+        array.insert(i + 1, deleted)
+      end
+    end
+  end
+  puts array
+end
+
+bubble_sort([10, 30, 55, 98, 44, 31])
```

Comment:
- [ ] Kindly remove the puts on the array and remove the test case also.

Reply:
Thank you very much for your review